### DISCUSSION TITLE
关于插件名称支持变量形式

### DISCUSSION THREAD

**Discussion by kibuniverse** [Tencent/bk-ci#6377] [Ideas]
期望通过插件名称配置变量的形式从而在运行时显示动态的内容
eg: 
    配置: 正在拉取 ${{branchName}} 代码
    运行：正在拉取 master 代码
具体的展示形式如果插件内容区域过小可以以hover 的形式展示

**Comment by zanyzhao** [comment]
看起来`nice to have`，喜欢楼主提议的请点vote
<img width="55" alt="image" src="https://user-images.githubusercontent.com/1629409/166876357-56d9e7cb-05c1-4940-86b0-2df528c2a46d.png">
